.agent/skills/i18n/scripts/i18n_sync.py

```
@@ -0,0 +1,163 @@
+
+    # Read base file
+    zh_cn_path = LOCALES_DIR / "zh-CN.json"
+    if not zh_cn_path.exists():
+        print(f"❌ Base file not found: {zh_cn_path}")
+        return
+
+    with open(zh_cn_path, "r", encoding="utf-8") as f:
+        zh_cn = json.load(f)
+
+    zh_cn_keys = get_all_keys(zh_cn)
+    print(f"📁 Base file: zh-CN.json")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)